CHANGELOG.md

```diff
@@ -1079,7 +1072,6 @@ This is a re-release of 1.22.0. Please refer to the 1.22.0 release notes for det
   df = copy(df)
   df.select(col("b").alias("c"))  # threw an error. Now it's fixed.
   ```
-
 - Fixed a bug in `Session.create_dataframe` that the non-nullable field in a schema is not respected for boolean type. Note that this fix is only effective when the user has the privilege to create a temp table.
 - Fixed a bug in SQL simplifier where non-select statements in `session.sql` dropped a SQL query when used with `limit()`.
 - Fixed a bug that raised an exception when session parameter `ERROR_ON_NONDETERMINISTIC_UPDATE` is true.
```

Comment:
there are too many changes in changelog. are they really necessary? can we roll them back?

Reply:
Those changes were already in main branch.

Reply:
I fixed it.